### ISSUE TITLE
Add Voicing to NumberPickers in MyChallengeAccordionBox

### ISSUE THREAD

**Issue by zepumph** [phetsims/ratio-and-proportion#441]
Related to https://github.com/phetsims/ratio-and-proportion/issues/381

https://docs.google.com/document/d/1offJbONUg8y5Aa9kHewC9iTDtNISzna4giGt7WrpXCk/edit#heading=h.jbqbxs39dkh5
https://github.com/phetsims/ratio-and-proportion/issues/363

**Comment by zepumph** [comment]
Ok. This is complete. Please review.

**Comment by terracoda** [comment]
I may have commented in the wrong issue for the design changes to the number spinners.

- [ ] Remove hint on keyboard focus. (Have a hint on the reading block).
- [ ] Use PDOM responses for all input.

![image](https://user-images.githubusercontent.com/1902116/158893873-1def44bd-dbb6-4281-883b-4ba1e7126f13.png)
![image](https://user-images.githubusercontent.com/1902116/158893918-42d2c752-f5d4-4c29-9aad-c35171ef3c9f.png)

**Comment by terracoda** [comment]
I found my original uncommitted comment. It was over in issue #440, but not posted.

**From #440:**
I have updated the interactions by input in the design doc.

For the number spinners, we will now use the default PDOM responses for all inputs. The custom hint has been moved to the all-input reading block in issue #440 . For pointer input, it means we won't hear Left Value and Right Value anywhere. These words are never part of the visual experience, so I am thinking this is just fine.

**Pointer Input:** object response & context response when actively pressing the number spinner arrows.
- 2, Current challenge 2 to 2. Hands far from new challenge ratio. 

**Alt Input:** on focus we still get name response & object response, but no longer a hint. The hint is part of the Reading Block, see #440 
- Left Value, 1
- Right Value, 2

**Active changes with keyboard** same responses as changes with pointer input.




Here's the design for the record.
![image](https://user-images.githubusercontent.com/1902116/158881621-fe8bc8c8-5e9c-4e9a-81dd-6aad4ce6c36f.png)
![image](https://user-images.githubusercontent.com/1902116/158881704-e8196087-2ae0-4f84-b8a4-dd9fcfff8852.png)
